Add missing inclusion header files. "geteuid" needs "sys/types.h" and
"unistd.h", otherwise, gcc 4.7 complains.

diff -urN ./languages/chewing/chewing_session_factory.cc.orig ./languages/chewing/chewing_session_factory.cc
--- ./languages/chewing/chewing_session_factory.cc.orig    2012-03-26 01:39:17.000000000 -0700
+++ ./languages/chewing/chewing_session_factory.cc    2012-04-17 17:36:21.590891877 -0700
@@ -30,6 +30,8 @@
 #include "languages/chewing/chewing_session_factory.h"
 
 #include <pwd.h>
+#include <sys/types.h>
+#include <unistd.h>
 
 #include "base/singleton.h"
 #include "base/util.h"

